examples/Viewer.tsx

```diff
@@ -394,19 +393,21 @@ class Viewer extends React.Component<{}, ViewerState> {
                             <label htmlFor={id}>{id}</label>
                             <input
                                 type="checkbox"
-                                onClick={(event) =>
-                                    this.turnAgentsOnOff(event.target.value)
-                                }
+                                onClick={(event) => {
+                                    const target: { value: string; }  = event.target as HTMLInputElement;
+                                    return this.turnAgentsOnOff(target.value)
+                                }}
                                 value={id}
                                 defaultChecked={true}
                             />
                             <input
                                 type="checkbox"
-                                onClick={(event) =>
+                                onClick={(event) => {
+                                    const target: { value: string; }  = event.target as HTMLInputElement;
```

Comment:
would it be more accurate to type `const target` as HTMLInputElement?

Reply:
Is it possible to just do `onClick={(event: React.MouseEvent<HTMLInputElement>) => {`?

Reply:
That sounds good. Also, I was even thinking 
```
this.turnAgentHighlightsOnOff((event.target as HTMLInputElement).value)
```

Reply:
Ugh 🤦🏻‍♀️ okay 😁 thanks for checking